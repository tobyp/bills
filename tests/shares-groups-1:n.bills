#!./bills
A	BCD.	3.00
