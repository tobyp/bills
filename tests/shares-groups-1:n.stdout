todo: B pays 1.0 to A
todo: C pays 1.0 to A
todo: D pays 1.0 to A
